stripe_test.go

```diff
@@ -38,6 +40,66 @@ func TestCheckinBackendConfigurationNewRequestWithStripeAccount(t *testing.T) {
 	}
 }
 
+func TestCheckinUserAgent(t *testing.T) {
```

Comment:
I also added a basic test for `User-Agent` because I want to use it on my next PR for `application` support.